clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -495,7 +496,14 @@ public ApiException build(String message) {
             public ApiException build(String message) {
                 return new OperationNotAttemptedException(message);
             }
-        });
+    }),
+    KAFKA_STORAGE_ERROR(56, "Disk error when trying to access log file on the disk.",
```

Comment:
The issue that I am thinking is the following. The 0.11.0 client jar doesn't have the error code for KafkaStorageException. If the 011.1 server sends a KafkaStorageException to the 0.11.0 client, the client will treat it as an UnknownServerException and won't retry. However, ideally, we want the 0.11.0 client to retry in this case.